apps/design-land/src/app/button/button.module.ts

```diff
@@ -31,28 +16,9 @@ import {
 
     DesignLandButtonRoutingModule,
     DesignLandExampleViewerModule,
-    DaffButtonSetModule,
-    DaffButtonModule,
     DaffArticleModule,
-    ButtonExamplesModule,
   ],
 })
 export class ButtonModule {
```

Comment:
Can we update this to `DesignLandButtonModule` and do it in the component too since you started doing that to other components in the PR?

Reply:
fixed.

Reply:
I've backed these changes out and will submit in another PR.